.travis.yml

```diff
@@ -1,15 +1,36 @@
 language: c
 compiler: gcc
-dist: xenial
-before_script:
-- pip install --user redis ramp-packer rmtest
-- git clone -b 4.0.1 --depth 1 https://github.com/antirez/redis.git
-- cd redis
-- make
-- cd ..
+dist: trusty
+
 env:
-- REDIS_MODULE_PATH=$TRAVIS_BUILD_DIR/src/libmodule.so REDIS_PATH=$TRAVIS_BUILD_DIR/redis/src/redis-server
+  - MODULE_NAME=redisgraph
+  - REDIS_MODULE_PATH=$TRAVIS_BUILD_DIR/src/libmodule.so
+  - REDIS_PATH=$TRAVIS_BUILD_DIR/redis/src/redis-server
+
+install:
+  - make clean && make
+
+before_script:
+  - pip install --user redis ramp-packer rmtest
+  - git clone -b 4.0.1 --depth 1 https://github.com/antirez/redis.git
+  - cd redis
+  - make
+  - cd ..
+
 script:
-- cmake .
-- make
-- make test
\ No newline at end of file
+  - cmake .
+  - make
+  - make test
+
+deploy:
+  # Module latest artifact
+  - provider: s3
+    access_key_id: $AWS_ACCESS_KEY
+    secret_access_key: $AWS_SECRET
```

Comment:
AWS_SECRECT isn't defined.

Reply:
The way redisearch uses access_key_id and secret_access_key in his .travis.yaml is incorrect.
The better way to use it is as it done in rejson and rebloom who take that data from those environment variables which must be declared in s3. This is why I have done it.